primitives/application-crypto/src/traits.rs

```diff
@@ -115,6 +115,8 @@ pub trait RuntimePublic: Sized {
 pub trait RuntimeAppPublic: Sized {
 	/// An identifier for this application-specific key type.
 	const ID: KeyTypeId;
+	/// An identifier for this application-specific crypto type.
```

Comment:
This struct is not an "application-specific crypto type".

```suggestion
	/// The identifier of the crypto type of this application-specific key type.
```